newsubprojects.md

```diff
@@ -0,0 +1,183 @@
+existing scope, activities and development of Project Jupyter. The process for direct Subproject
+creation is minimal and informal: Steering Council members should be in consensus about the
+Subproject's creation and notify the main Jupyter list of its creation.
+
+
+## Incorporation of an existing, external subproject
+
+In other cases, Subprojects proposed for incorporation will have existed for some time outside the
+official Jupyter organization. This section describes the incorporation process for these existing,
+external subprojects. This process applies to Subprojects that have incubated under the
+`jupyter-incubatror` GitHub organization (see below), Subprojects developed under other GitHub
+organizations or even Subprojects developed in other public version control systems.
+
+When a Subproject is incorporated, it becomes an officialy supported and maintained part
+of Project Jupter and is moved to the [jupyter](https://github.com/jupyter) GitHub organization.
+
+
+### Proposal for incorporation
+
+For an existing, external Subproject to be incorporated into the main Jupyter organization, the
+following proposal process will be used:
+
+1. The Subproject team should submit a pull request against the
+   [jupyter/enhancement-proposals](https://github.com/jupyter/enhancement-proposals) repository
+   with an enhancement proposal for including the subproject in the main Jupyter organization. 
+   The enchancement proposal should describe how the Subproject meets each of the above criteria.
+2. The proposal for incorporation will be discussed by the community using that pull request.
+3. A recommendation will be made by the consensus of the Steering Council.
+
+The possible recommendations of the Steering Council wil be:
+
+* Integration into an existing official Suproject.
+* Incorporation as a new official Subproject.
+* Further internal or external incubation (see below) with recommendations on steps the team can
+  take to prepare for future incorporation.
+* Rejection. This option will be used if the Steering Council thinks the proposed Subproject is not
+  progressing or will never be appropriate for graduation. If a rejected Subproject has been
+  incubating under the [jupyter-incubator](https://github.com/jupyter-incubator) GitHub
+  organization, its repository will be removed from that organization after a transition period.
+
+
+### Incorporation
+
+When an existing, external Subproject is incorporated as a new official Subproject, the following
+steps will be taken:
+
+1. The repository will be transfered over to the main Jupyter GitHub organization.
+2. A GitHub team will be created for the Subproject, with the Subproject team having
+   read/write permissions on the Subproject repository.
+3. The team will send an Email the main Jupyter list with an announcement about the new
+   Subproject.
+4. The standard Jupyter LICENSE file will be added to the repository.
```

Comment:
licenses are also in headers of files, etc.  Should we cover those?  If I have a file that says "copyright by Jason Grout", does that need to change to be IPython developers or something?